src/linux/system.rs

```diff
@@ -566,9 +566,13 @@ fn get_all_disks() -> Vec<Disk> {
             //
             // In some other cases, it uses a device mapper to map physical block devices onto
             // higher-level virtual block devices (on `/dev/mapper`).
+            //
+            // Raspbian uses root and mmcblk for physical disks
             line.starts_with("/dev/sd") ||
             line.starts_with("/dev/nvme") ||
             line.starts_with("/dev/mapper/")
```

Comment:
You forgot to add `||` at the end of this line. ;)

Reply:
woops, don't know how that slipped by